handbook/digital-experience/README.md

```diff
@@ -354,22 +354,23 @@ The Apprentice schedules all travel arrangements for the CEO including flights,
 ### Process incoming equipment
 Upon receiving any device, the Apprentice will process the incoming equipment by:
 1. Search for the SN of the physical device in the ["Company equipment" spreadsheet](https://docs.google.com/spreadsheets/d/1hFlymLlRWIaWeVh14IRz03yE-ytBLfUaqVz0VVmmoGI/edit#gid=0) to confirm the correct equipment was received.
-  - If the serial numbers do not match [create an issue](https://fleetdm.com/handbook/business-operations#contact-us) to get help from the Business Operations department. 
 3. Visibly inspect equipment and all related components (e.g. laptop charger) for damage.
 4. Remove any stickers and clean devices and components.
 5. Using the device's charger plug in the device.
 6. Turn on the device and enter recovery mode using the [appropriate method](https://support.apple.com/en-us/HT204904).
 7. Connect the device to WIFI.
 8. Using the "Recovery assistant" tab (In the top left corner), select "Delete this Mac".
 9. Follow the prompts to activate the device and reinstall the appropriate version of macOS.
+> If you are prevented from completing the steps above, create a [hardware issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-business-operations&projects=&template=hardware-issue.md&title=Hardware+or+software+issue) for IT, for the device to be scheduled for troubleshooting and remediation. Please note in the issue where you encountered blockers to completing the steps.
 
 
 ### Ship approved equipment
 Once the Business Operations department approves inventory to be shipped from Fleet IT, the Apprentice will ship the equipment by:
 1. Compare the equipment request issue with the ["Company equipment" spreadsheet](https://docs.google.com/spreadsheets/d/1hFlymLlRWIaWeVh14IRz03yE-ytBLfUaqVz0VVmmoGI/edit#gid=0) and verify physical inventory.
 2. Plug in the device and ensure inventory has been correctly processed and all components are present (e.g. charger cord, power converter).
-3. package equipment for shipment and include Yubikeys (if requested).
-4. Change the "Company equipment" spreadsheet to reflect the new user
+    - If the device is not processed correctly, repeat the [process incoming equipment steps](https://fleetdm.com/handbook/digital-experience#process-incoming-equipment). If you encounter any issues when following those steps, create a [hardware issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-business-operations&projects=&template=hardware-issue.md&title=Hardware+or+software+issue) for IT to troubleshoot the device.
```

Comment:
```suggestion
```
Removing for now until we can correct issue templates.

Reply:
@Sampfluger88 what's wrong with the issue templates?